concepticondata/references/references.bib

```diff
@@ -4266,5 +4266,14 @@ @Article{Nordenskioeld1905
   Number                              = {3},
   Journal                             = {Ymer},
   Title                               = {Beiträge zur Kenntnis einiger Indianerstämme des Rio Madre de Dios-gebietes [Contributions to the Knowledge about some Indian tribes of the area of the Rio Madre de Dios]},
-  Year                                = {1905},
+  Year                                = {1905}
+}
+
+@Book{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
+  Title                               = {West Old Turkic},
+  Subtitle                            = {Turkic loanwords in Hungarian}
```

Comment:
We don't use the submtitle category in Concepticon. Just add the subtitle with a dot to the title.

Reply:
Resolved in c6dbb41d1118eab6a1571917fa1d6db0b9a01c7c
Also fixed a missing comma.